cid/helpers/glue.py

```diff
@@ -32,3 +36,54 @@ def delete_table(self, name, catalog, database):
             )
         except self.client.exceptions.EntityNotFoundException:
             return True
+
+    def create_database(self, name: str, catalog_id: str) -> None:
+        """Create a new AWS Glue database"""
+        try:
+            self.client.create_database(
+                CatalogId=catalog_id,
+                DatabaseInput={
+                    "Name": name,
+                },
+                Tags=self.default_tag_dict
+            )
+        except self.client.exceptions.AlreadyExistsException as ex:
+            logger.debug(ex, exc_info=True)
+            logger.info("Database %s cannot be created in %s: %s", name, catalog_id, ex)
+            raise CidError() from ex
```

Comment:
In other paces we use a logic of 'ensure' this means if we ask to create it will create or return exisitng. Simiarly with delete if the final state is achieved we do not need to raise a problem. 

Please can you review if this fits?